modules/ztp-syncing-new-configmap-changes-to-existing-pgt-crs.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-syncing-new-configmap-changes-to-existing-pgt-crs_{context}"]
+= Syncing new ConfigMap changes to existing PolicyGenTemplate CRs
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have created a `PolicyGenTemplate` CR that pulls information from a `ConfigMap` CR using hub cluster templates.
+
+.Procedure
+
+. Update the contents of your `ConfigMap` CR, and apply the changes in the hub cluster.
+
+. To sync the contents of the updated `ConfigMap` CR to the deployed policy, do either of the following:
+
+.. Optional: Delete the existing policy and recreate it using the `PolicyGenTemplate` CR.
+
+... Delete the policy, for example:
++
+[source,terminal]
+----
+$ oc delete policy <policy_name> -n <policy_namespace>
+----
++
+... Re-apply the `PolicyGenTemplate` CR:
++
+[source,terminal]
+----
+$ oc apply -f sriov-pgt-example.yaml
```

Comment:
Should be removed. ArgoCD will automatically recreate the policy when the policy is deleted by the above delete command.